src/segments/firebase.go

```diff
@@ -53,7 +54,7 @@ func (f *Firebase) Enabled() bool {
 	// Test if the current directory is a project path
 	// and if it is, return the project name
 	for key, value := range data.ActiveProject {
-		if key == f.env.Pwd() {
+		if key == f.env.Pwd() || strings.HasPrefix(f.env.Pwd(), key) {
```

Comment:
Can't this only be the HasPrefix check? That includes equals, no?

Reply:
You're absolutely correct! Ive updated the conditional to just the `HasPrefix()` call